io.gdevs.GDLauncher.yml

```diff
@@ -0,0 +1,54 @@
+app-id: io.gdevs.GDLauncher
+base: "org.electronjs.Electron2.BaseApp"
+base-version: "20.08"
+runtime: "org.freedesktop.Sdk"
+runtime-version: "20.08"
+sdk: "org.freedesktop.Sdk"
+sdk-extensions:
+    - org.freedesktop.Sdk.Extension.openjdk
+command: gd.sh
+build-options:
+    env:
+        PATH: /app/jre/bin:/usr/bin
+finish-args:
+    - --socket=x11
+    - --share=ipc
+    - --device=all # controller support
+    - --socket=pulseaudio
+    - --share=network
+    - --env=CHROME_WRAPPER=zypak-wrapper
+modules:
+    - name: openjdk
+      buildsystem: simple
+      build-commands:
+        - /usr/lib/sdk/openjdk/install.sh
+    - name: gdlauncher
+      buildsystem: simple
+      build-commands:
+        - chmod +x GDLauncher-linux-setup.AppImage
+        - ./GDLauncher-linux-setup.AppImage --appimage-extract
+        - cp -r squashfs-root /app/bin/gdlauncher
+        - install -D gdlauncherscaled.png /app/share/icons/hicolor/512x512/apps/io.gdevs.GDLauncher.png
+        - install -D io.gdevs.GDLauncher.desktop /app/share/applications/io.gdevs.GDLauncher.desktop
+        - install -D io.gdevs.GDLauncher.metainfo.xml /app/share/metainfo/io.gdevs.GDLauncher.appdata.xml
+      sources:
+          - type: file
+            path: GDLauncher-linux-setup
+            url: https://github.com/gorilla-devs/GDLauncher/releases/latest/download/GDLauncher-linux-setup.AppImage
+            sha256: 83192422964903efe4a9a84b995861f506a9f47252bb8cc45f5fdd99939ba09a
+          - type: file
+            path: io.gdevs.GDLauncher.desktop
+          - type: file
+            path: gdlauncherscaled.png
+          - type: file
+            path: io.gdevs.GDLauncher.metainfo.xml
+
+    - name: gd.sh
+      buildsystem: simple
+      build-commands:
+        - install -D gd.sh /app/bin
```

Comment:
Don't this install `gd.sh` as `/app/bin` ?

Reply:
Not on my system, no.